### ISSUE TITLE
missing module tifffile

### ISSUE THREAD

**Issue by FabianIsensee** [ilastik/ilastik-build-conda#10]
I installed ilastik as described in the readme. After activating the environment and attempting to run ilastik using run_ilastik.sh I get the following error message:

```
  File "/home/fabian/miniconda/envs/ilastik-devel/ilastik-meta/lazyflow/lazyflow/operators/ioOperators/opTiffReader.py", line 8, in <module>
    import tifffile
ImportError: No module named tifffile
```

Apparently, not all required packages are installed.

My system is a ubuntu 14.04

Cheers,
Fabian

**Comment by stuarteberg** [comment]
What is the output of `conda info --root` and `conda list`?  And what if you install `tifffile` manually via `conda install -c ilastik tifffile`?

**Comment by FabianIsensee** [comment]
Hi Stuart,
What is the output of conda info --root and conda list?
=> /home/fabian/miniconda

And what if you install tifffile manually via conda install -c
ilastik tifffile?
The following packages will be downloaded:

| package | build |
| --- | --- |
| numpy-1.10.1 | py27_0         5.9 MB |

The following NEW packages will be INSTALLED:

```
tifffile: 0.4.1-1
```

The following packages will be UPDATED:
numpy:    1.9.3-py27_0 --> 1.10.1-py27_0

Proceed ([y]/n)?

It will prompt me to update the numpy version in the evironment. Not sure
whether this will break additional things. Do you think it is safe to do it
anyways?

Cheers,

Fabian

2015-10-22 13:33 GMT+02:00 Stuart Berg notifications@github.com:

**Comment by stuarteberg** [comment]
> It will prompt me to update the numpy version in the evironment. Not sure whether this will break additional things.

If it does break something, you can always start over with a fresh environment (it won't need to re-download all the packages again, so starting over will be quick).

> Do you think it is safe to do it anyways?

It's hard to know without seeing the output from `conda list`, but I think it should be fine.

**Comment by stuarteberg** [comment]
OK, the continuous integration server started having the same problem:
https://circleci.com/gh/ilastik/lazyflow/123

From what I can see `conda` is making an error in its dependency resolution.  Conda is choosing to install the wrong build of our metapackage named `ilastik-deps-pc-headless`.  Instead of choosing the latest build (`build-number=2`):

`linux-64/ilastik-deps-pc-headless-1.1-np19py27_2.tar.bz2`,

it is choosing an old build (`build-number=0`):

`linux-64/ilastik-deps-pc-headless-1.1-np19py27_0.tar.bz2`

As it turns out, the `tifffile` dependency is not listed in build 0, so that's why it's missing on your machine.

As a workaround, I've simply removed **all** old builds of our packages from the `main` ilastik channel (anaconda.org/ilastik).  Now conda has only one choice when installing `ilastik-deps-pc-headless`.

Long story short: This should be fixed now.
